doc.go

```diff
@@ -22,128 +22,172 @@ Usage:
+                               creating a block (default: 375000)
+      --blockprioritysize=     Size in bytes for high-priority/low-fee
+                               transactions when creating a block (default:
+                               20000)
+      --miningtimeoffset=      Offset the mining timestamp of a block by this
+                               many seconds (positive values are in the past)
+      --nonaggressive          Disable mining off of the parent block of the
+                               blockchain if there aren't enough voters
+      --nominingstatesync      Disable synchronizing the mining state with other
+                               nodes
+      --allowunsyncedmining    Allow block templates to be generated even when
+                               the chain is not considered synced on networks
+                               other than the main network.  This is
+                               automatically enabled when the simnet option is
+                               set.  Don't do this unless you know what you're
+                               doing
+      --txindex                Maintain a full hash-based transaction index
+                               which makes all transactions available via the
+                               getrawtransaction RPC
+      --droptxindex            Deletes the hash-based transaction index from the
+                               database on start up and then exits
+      --addrindex              Maintain a full address-based transaction index
+                               which makes the searchrawtransactions RPC
+                               available
+      --dropaddrindex          Deletes the address-based transaction index from
+                               the database on start up and then exits
+      --noexistsaddrindex      Disable the exists address index, which tracks
+                               whether or not an address has even been used
+      --dropexistsaddrindex    Deletes the exists address index from the
+                               database on start up and then exits
+      --nocfilters             (Deprecated) Disable compact filtering (CF)
+                               support
+      --dropcfindex            (Deprecated) Deletes the index used for compact
+                               filtering (CF) support from the database on start
+                               up and then exits
+      --piperx=                File descriptor of read end pipe to enable parent
+                               -> child process communication
+      --pipetx=                File descriptor of write end pipe to enable
+                               parent <- child process communication
+      --lifetimeevents         Send lifetime notifications over the TX pipe
 
 Help Options:
-  -h, --help           Show this help message
+  -?                           Show this help message
```

Comment:
I can't seem to make this work:
```
$ ./dcrd -?
unknown flag `?'
```

Reply:
It works on Windows, but I suppose the docs should leave it out since everything else works on both platforms and this doesn't include the Windows service options either.  Removed.